scss/modules/_grid.scss

```diff
@@ -9,38 +9,73 @@ $grid-col-padding: 15px;
   .container-fluid { width: 100%; }
 
   .container {
+    margin: 0 auto;
     max-width: $site-max-width;
-    margin-right: auto;
-    margin-left: auto;
   }
 
   .row {
-    @include clearfix-overflow-visible;
-    margin-right: -$grid-col-padding;
-    margin-left: -$grid-col-padding;
+    margin-right: auto;
+    margin-left: auto;
+    
+    &:before,
+    &:after,
+    & &:before,
+    & &:after  {
+      content: " ";
+      display: table;
+    }
+
+    &:after,
+    & &:after {
+      clear: both;
+    }
+
+    & & {
```

Comment:
Yes, I'm targeting rows that are nested in other rows. Will add comments to explain.